### PR TITLE
feat: post endpoint best practices

### LINKED ISSUE #19: Agrochemical companies should be able to post information on organic farming best practices
Agrochemicals companies should provide best practices information to farmers on how to practice organic farming effectively, this information will be based on certain categories. The details to capture include the following:

- post title.
- category(i.e wheat farming, maize, carrots).
- Author(reference).
- company(reference).
- detailed description
- best seeds to use(can be comma-separated values).

### PR DESCRIPTION
**Pull request by obsessedyouth**
## Description
 An end point such that when its called a POST request is sent to the db. Creation of a model for this data based on the issue and a route to the db.

Fixes #19 

## How Has This Been Tested?
Tested using a local mongodb server and postman for serving requests

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have added necessary inline code documentation
- [ ] I have added tests that prove my fix is effective and that this feature works
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: src/routes/bestPractice.router.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import { Router } from 'express';
+import createBestPractice from '../controllers/bestPractices.controller';
+
+const router = Router();
+
+router.post('/submit-form', (req, res, next) => {
+  createBestPractice({ ...req.body });
+  res.json({ success: 'action posted' });
+  next();
```
Comment (francis-mwas) [inline src/routes/bestPractice.router.js:9]: Nice job, but handle all the application logic on the controller, and let the router to only deals with routing.

Comment (obsessedyouth) [reply src/routes/bestPractice.router.js:9]: Thanks. I'll move the logic to the controller and it'll be an import only in the router.

Comment (francis-mwas) [reply src/routes/bestPractice.router.js:9]: yeah, exactly

Comment (obsessedyouth) [reply src/routes/bestPractice.router.js]: Fixed.

**Pull request merged**